src/frontend/app/features/cloud-foundry/users/manage-users/manage-users-modify/manage-users-modify.component.ts

```diff
@@ -122,8 +122,14 @@ export class UsersRolesModifyComponent implements OnInit, OnDestroy {
       map(org => [org]),
       share()
     );
+
+    const isTableLoading$ = orgConnect$.pipe(
```

Comment:
We should use the entityService from the fetchOrg method to work out if the org is being fetched. Lets us do error handling too.